example/src/modules/forms/TextAreaScreen.tsx

```diff
@@ -0,0 +1,81 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Icon,
+  Radio,
+  RadioGroup,
+  Slot,
+  Switch,
+  TextArea,
+  TextAreaSizes,
+  TextAreaVariants,
+  useTheme,
+} from "@adaptui/react-native-tailwind";
+
+export const TextAreaScreen = () => {
+  const tailwind = useTheme();
+  const [variant, setVariant] = useState("subtle");
```

Comment:
Use types here than using `as <type>` while sending it as props to the component.